src/index.js

```diff
@@ -28,26 +43,27 @@ module.exports = function purescriptLoader(source, map) {
     pscIde: false,
     pscIdeColors: webpackOptions.psc === 'psa' || query.psc === 'psa',
     pscIdeArgs: {},
+    pscPackage: false,
     bundleOutput: 'output/bundle.js',
     bundleNamespace: 'PS',
     bundle: false,
     warnings: true,
     output: 'output',
     src: [
       path.join('src', '**', '*.purs'),
-      path.join('bower_components', 'purescript-*', 'src', '**', '*.purs')
+      ...depsPaths(query.pscPackage)
```

Comment:
Does `npm run build` work with the spread operator? We just have `es2015` as the preset in the `.babelrc`. Do we need a babel plugin to support this?